sormas-app/app/src/main/java/de/symeda/sormas/app/util/DiseaseConfigurationCache.java

```diff
@@ -76,9 +70,12 @@ private DiseaseConfigurationCache() {
 			} else {
 				nonPrimaryDiseases.add(disease);
 			}
-			if (Boolean.TRUE.equals(configuration.getCaseBased()) || (configuration.getCaseBased() == null && disease.isDefaultCaseBased())) {
+			if (Boolean.TRUE.equals(configuration.getCaseSurveillanceEnabled())
+				|| (configuration.getCaseSurveillanceEnabled() == null && disease.isDefaultCaseBased())) {
```

Comment:
`isDefaultCaseBased` should be renamed to `isDefaultCaseSurveillanceEnabled`

Reply:
updated